common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -96,7 +96,7 @@ protected[core] case class BlackBoxExec(image: String) extends Exec(Exec.BLACKBO
 /**
  * add temporary field that holds the "fixed" names for components where the '_' is replaced by the user's namespace
  */
-protected[core] case class SequenceExec(code: String, components: Vector[String], fixedComponents: Option[Vector[String]] = None) extends Exec(Exec.SEQUENCE) {
+protected[core] case class SequenceExec(code: String, components: Vector[String], fixedComponents: Option[Vector[FullyQualifiedEntityName]] = None) extends Exec(Exec.SEQUENCE) {
```

Comment:
I don't think they are really needed in the implementation. However, some tests that Perry added for the current implementation verify the names. I could try to modify those tests such that they pass with this change.